lib/boku/client.py

```diff
@@ -328,10 +339,25 @@ def _get(self, url):
         with statsd.timer('solitude.boku.api'):
             return requests.get(proxy)
 
+    def check_sig(self, data):
+        """
+        Check that a signature is valid. This check has to be done on the
+        proxy, because the solitude database does not have access to the
+        required BOKU_SECRET_KEY.
+        """
+        try:
+            self.api_call('/check_sig', data, add_timestamp=False)
```

Comment:
why does this need to make a fake API call? Can't you just move [this code](https://github.com/andymckay/solitude/blob/1130213/lib/proxy/views.py#L242-L255) into this function?